src/notificationcenter.cs

```diff
@@ -61,9 +67,45 @@ interface UIVibrancyEffect_NotificationCenter {
 #else
 		[EditorBrowsable (EditorBrowsableState.Advanced)] // this is not the one we want to be seen (compat only)
 #endif
+		[Deprecated (PlatformName.iOS, 10,0)]
 		[Static, Export ("notificationCenterVibrancyEffect")]
 		UIVibrancyEffect NotificationCenterVibrancyEffect ();
 	}
+
+	[Category]
+	[BaseType (typeof (NSExtensionContext))]
+	interface NSExtensionContext_NCWidgetAdditions {
+		[iOS (10,0)]
+		[Export ("widgetLargestAvailableDisplayMode")]
+		NCWidgetDisplayMode GetWidgetLargestAvailableDisplayMode ();
+
+		[iOS (10,0)]
+		[Export ("setWidgetLargestAvailableDisplayMode:")]
+		void SetWidgetLargestAvailableDisplayMode (NCWidgetDisplayMode mode);
+
+		[iOS (10,0)]
+		[Export ("widgetActiveDisplayMode", ArgumentSemantic.Assign)]
```

Comment:
No we don't. It was a property that needed to be converted (because it's a `[Category]`) and I forgot to remove the `ArgumentSemantic`. Good catch!